AzureCalling/External/Calling/CallingContext.swift

```diff
@@ -79,29 +79,39 @@ class CallingContext: NSObject {
             let joinCallOptions = JoinCallOptions()
 
             if joinConfig.isCameraOn && self.localVideoStream != nil {
-                let videoOptions = VideoOptions(localVideoStream: self.localVideoStream)
-                joinCallOptions!.videoOptions = videoOptions
+                let localVideoStreamArray = [self.localVideoStream!]
+                let videoOptions = VideoOptions(localVideoStreams: localVideoStreamArray)
+                joinCallOptions.videoOptions = videoOptions
             }
 
-            joinCallOptions!.audioOptions = AudioOptions()
-            joinCallOptions!.audioOptions.muted = joinConfig.isMicrophoneMuted
+            joinCallOptions.audioOptions = AudioOptions()
+            joinCallOptions.audioOptions?.muted = joinConfig.isMicrophoneMuted
```

Comment:
Is this correct to have it optional?

Reply:
Should be correct. The audioOptions from joinCallOptions is optional, use `?` instead of `!` could be more safe.